authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
+
+* An OAuth access token - Users can get an OAuth token from:
++
+ ** An `Authorization: Bearer…​` header.
+ ** A WebSocket subprotocol header for the web socket requests.
+ ** The built-in OAuth server through specific endpoints.
++
+Users can also xref:../authentication/managing-oauth-access-tokens.adoc#managing-oauth-access-tokens[view and manage their own OAuth acces tokens].
+
+* An X.509 client certificate - You require an HTTPS connection to the API server to use the X.509 client certificates for authentication.
+
+[NOTE]
+====
+If you do not present a valid access token or certificate, your request is unauthenticated and you receive an HTTP 401 error.
+====
+An administrator can configure authentication through the following tasks:
+
+* Configure an identity provider: You can define any xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[supported identity provider in {product-title}] and add it to your cluster. To specify an identity provider, you must create a custom resource (CR) that describes the identity provider.
+* xref:../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[Configure the internal OAuth server]: The {product-title} master includes a built-in OAuth server that determines the user’s identity from the configured identity provider and creates access tokens. You can configure the token duration and inactivity timeout, and customize the internal OAuth server URL.
+* xref:../authentication/configuring-oauth-clients.adoc#oauth-register-additional-client_configuring-oauth-clients[Register an OAuth client]: {product-title} includes many xref:../authentication/configuring-oauth-clients.adoc#oauth-default-clients_configuring-oauth-clients[default OAuth clients]. You can register and configure additional OAuth clients.
++
+[NOTE]
+====
+Users must specify either a default or custom OAuth client that receives and uses the token in their requests for the OAuth token.
+====
+
+* Manage cloud provider credentials using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[Cloud Credentials Operator].
+
+[id="authorization-overview"]
+== Overview of authorization in {product-title}
+Authorization involves determining whether a user has the required permissions to perform an action.
+
+Based on how users interact with an {product-title} cluster, users can be of xref:../authentication/understanding-authentication.adoc#rbac-users_understanding-authentication[three types] - regular users, system users, and service accounts. By default, each user type has a set of access permissions and privileges. These permissions and privileges are represented through the xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[role-based access control (RBAC) objects], such as rules, roles, and bindings.
```

Comment:
Suggest a colon (:) instead of a dash (-) before you list the 3 types of users.

Also suggest rewording "users can be of three types", which sounds a little off to me. Maybe you could rearrange to starting with "There are three types of users... " ?